examples/server.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//! This example sets up a local Drawbridge server on port 12345.
+//! Run this example with `cargo run --example server`.
+
+use std::net::Ipv4Addr;
+
+use drawbridge_app::Builder;
+use hyper::Server;
+use tempfile::tempdir;
+
+#[tokio::main]
+async fn main() {
+    let store = tempdir().unwrap();
+
+    let service = Builder::new(store.path()).build().unwrap();
+
+    let socket = (Ipv4Addr::LOCALHOST, 12345).into();
```

Comment:
Either run on the same port as production instance, or run on `:0` and print the value on startup